fastlane/lib/fastlane/helper/gradle_helper.rb

```diff
@@ -48,7 +48,7 @@ def load_all_tasks
         self.tasks = []
 
         command = [escaped_gradle_path, "tasks", "--console=plain"].join(" ")
-        output = Actions.sh(command, log: false)
+        output = Action.sh(command, print_command: false, print_command_output: false)
```

Comment:
The original implementation of `Action.sh` does not accept any parameters other than the command, so it was not possible to control the command output using it. That was likely the original reason for choosing Actions in this case.

Because we've extended `Action.sh` to now have those optional named parameters, it is now suitable for the control that was needed.

Overall we felt it was better not to have the same code calling both `Action.sh` and `Actions.sh` (even though one calls the other as implementation)